dlpython/binding.go

```diff
@@ -296,14 +292,10 @@ func mapCalls() error {
 	defer C.free(unsafe.Pointer(s_Py_DecRef))
 	C.Py_DecRef_ptr = C.Py_DecRef_f(C.dlsym(C.python_lib, s_Py_DecRef))
 
-	s_PyTuple_ClearFreeList := C.CString("PyTuple_ClearFreeList")
```

Comment:
@matiasinsaurralde if we delete this aren't we breaking backwards compatibility with the old versions of py which use `PyTuple_ClearFreeList`?

Reply:
wouldn't it better to start receiving the python version in `mapCalls` and ignore this code for >3.8 versions?

Reply:
@tbuchaillot We actually don't use `PyTuple_ClearFreeList` in any of the codebase, but as it was removed in one of the Python 3.x., the map call was causing issues. It's safe to remove it.

Reply:
I think passing the version to `mapCalls` is interesting but might not be required at this. time, in case [Python 4 ever lands](https://medium.com/codex/python-4-0-will-never-arrive-3d994dce54f1) and breaks any mapping, we can incorporate it 👍